packages/api-explorer/package.json

```diff
@@ -17,7 +17,7 @@
     "fetch-har": "^4.0.2",
     "js-cookie": "^2.1.4",
     "lodash.kebabcase": "^4.1.1",
-    "oas": "^4.0.0",
+    "oas": "^4.1.0",
```

Comment:
I messed up and tagged this breaking change within `oas` to 4.1.0 instead of 5.0.0 so this PR might not work locally until I update this.